### PR TITLE
feat: fix home add token and new seed bug

### PR DESCRIPTION
**Pull request by RickStormBringer**
1.refresh token status  when token is added.
2.fix new seed route jump bug
3.PORTAL-1381
4.PORTAL-1388
5.PORTAL-1293
6.PORTAL-1294

---
This change is [Reviewable](https://reviewable.io/reviews/conflux-chain/helios/209)

### CODE REVIEW

File: packages/popup/src/pages/Home/components/AddToken.js [hunk 1]
```diff
@@ -89,6 +101,9 @@ function AddToken({onClose, onOpen}) {
       if (result) {
         mutate([WALLETDB_REFETCH_BALANCE])
         mutate([WALLETDB_ADD_TOKEN_LIST])
```
Comment (yqrashawn) [inline packages/popup/src/pages/Home/components/AddToken.js:103]: the second mutate should be invoked after the first mutate resolved

```js
mutate([WALLETDB_REFETCH_BALANCE]).then(() => mutate([WALLETDB_ADD_TOKEN_LIST]))
```

**Review by angelia-yuqi-personal — DISMISSED** [review]

**Review by yqrashawn — CHANGES_REQUESTED** [review]

File: packages/popup/src/pages/Home/components/AddToken.js (hunk 1 above)
Comment (RickStormBringer) [reply packages/popup/src/pages/Home/components/AddToken.js]: done

**Comment by angelia-yuqi-personal** [comment]
Task linked: [PORTAL-1381 修复 Bug: ：接口。wallet_watchAsset. image 可否改为可选参数](https://app.clickup.com/t/3719212/PORTAL-1381)

**Comment by angelia-yuqi-personal** [comment]
Task linked: [PORTAL-1294 修复 Bug: My accounts -Add account 导入登录账户自己的助记词报错JSON-RPC InvalidParams -32602](https://app.clickup.com/t/3719212/PORTAL-1294)

**Comment by angelia-yuqi-personal** [comment]
Task linked: [PORTAL-1293 修复 Bug: 导入重复的私钥报错 JSON-RPC InvalidParams -32602](https://app.clickup.com/t/3719212/PORTAL-1293)

**Comment by angelia-yuqi-personal** [comment]
Task linked: [PORTAL-1388 验证 Bug: wallet_importMnemonic。在账户已有助记词的导入另外一组助记词的情况下报错 是否修复](https://app.clickup.com/t/3719212/PORTAL-1388)

**Review by angelia-yuqi-personal — APPROVED** [review]

**Review by yqrashawn — APPROVED** [review]

**Pull request merged**